CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+### citus v12.1.0 (September 12, 2023) ###
+
+* Adds PostgreSQL 16 RC1 support (#7173)
+
+* Add `citus_schema_move()` function which moves tables within a
+  distributed schema to another node (#7180)
+
+* Adds `citus_pause_node()` UDF that allows pausing the node with given id
+  (#7089)
+
+* Makes sure to enforce shard level colocation with the GUC
+  `citus.enable_non_colocated_router_query_pushdown` (#7076)
+
+* Allows creating reference / distributed-schema tables from local tables added
+  to metadata and that use identity columns (#7131)
+
+* Propagates `BUFFER_USAGE_LIMIT` option in `VACUUM` and `ANALYZE` (#7114)
+
+* Propagates `PROCESS_MAIN`, `SKIP_DATABASE_STATS`, `ONLY_DATABASE_STATS`
+  options in `VACUUM` (#7114)
+
+* Propagates `GENERIC_PLAN` option in `EXPLAIN` (#7141)
+
+* Propagates "rules" option in `CREATE COLLATION` (#7185)
+
+* Propagates `GRANT`/ `REVOKE` for database privileges (#7109)
+
+* Adds citus_truncate_trigger to Citus foreign tables (#7170)
+
+* Removes `pg_send_cancellation` (#7135)
+
+* Prevents unnecessarily pulling the data into coordinator for some
+  `INSERT .. SELECT` queries that target a single-shard group (#7077)
+
+* Makes sure that rebalancer throws an error if replication factor is greater
+  than the shard allowed node count. Also makes sure to avoid moving a shard
+  to a node that it already exists on (#7074)
```

Comment:
```suggestion
* Makes sure that rebalancer throws an error if replication factor is greater
  than the shard allowed node count. Also makes sure to avoid moving a shard
  to a node that it already exists on. (#7074)
```